pkgs/development/libraries/libewf-legacy/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, fetchurl
+, fetchpatch
+, stdenv
+, zlib
+, openssl
+, libuuid
+, pkg-config
+, bzip2
+}:
+
+stdenv.mkDerivation rec {
+  version = "20140814";
+  pname = "libewf-ewf";
+
+  src = fetchurl {
+    url = "https://github.com/libyal/libewf-legacy/releases/download/${version}/libewf-${version}.tar.gz";
```

Comment:
Can we use the github auto generated tarball instead? What's the difference?

Reply:
I followed the same "style" of [libewf](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/libewf/default.nix) file.

Reply:
Maybe the original expressions for libewf  was too old.

Reply:
The difference is the configure script being pregenerated, and the original-original expression is older than `fetchFromGitHub` (or GitHub migration of `libewf`), only getting the minimal unavoidable changes.